info/v1/machine.go

```diff
@@ -176,6 +176,8 @@ type MachineInfo struct {
 	// Memory capacity and number of DIMMs by memory type
 	MemoryByType map[string]*MemoryInfo `json:"memory_by_type"`
 
+	NVMInfo NVMInfo `json:"nvm_info"`
```

Comment:
Please add a comment here

Reply:
Now reponse on /api/v2.0/machine request has following form:
`{
  ...
  "nvm_info": {
    "avg_power_budget": 0
  },
  "hugepages": [
    {
      "page_size": 1048576,
      "num_pages": 0
    },
    {
      "page_size": 2048,
      "num_pages": 2
    }
  ],`
  ...
}
I would like to see:
```
 "nvm": {
    "avg_power_budget": 0
  },
```
to have similar form as it is for the rest of keys on API (e.g. hugepages).

Reply:
Done!